### PR TITLE
Simplify function signature of add_kicktable

### PR DESCRIPTION
**Pull request by xresende**

### CODE REVIEW

File: src/flat_file.cpp [hunk 1]
```diff
@@ -392,11 +392,11 @@ static Status::type read_flat_file_tracy(const std::string& filename, Accelerato
         e.pass_method = PassMethod::pm_kickmap_pass;
         double tmpdbl; std::string filename;
         fp >> tmpdbl >> tmpdbl >> filename;
-        Status::type status = add_kicktable(filename, kicktable_list, e.kicktable_idx);
-        if (status == Status::success) {
+        unsigned int idx = add_kicktable(filename);
+        if (idx < 0) {
```
Comment (anacso17) [inline src/flat_file.cpp:396]: is this condition correct?

Comment (xresende) [reply src/flat_file.cpp:396]: the condition is correct, @anacso17 . but your comment reminded that using 'unsigned int' as return type of add_kicktable is no appropriate: it turns out that c++ compiler treats unsigned int and int the same, for comparison reasons with other variable and with literals. only the bit representation changes.  for example, the following code works!

but  ```int``` instead of ```unsigned int``` is the correct type. i'll change. thanks!

```
#include <iostream>

int main() {
    int a = -1;
    unsigned int b = -1;
    if (b == -1) {
        std::cout << b << std::endl;
    }
    if (b == a) {
        std::cout << b << std::endl;
    }
}```

Comment (xresende) [reply src/flat_file.cpp]: you are right. the conditional at the Tracy/AT flat_file reader for kicktables is inverted. fixed!

**Review by anacso17 — APPROVED** [review]

**Review by fernandohds564 — APPROVED** [review]

**Review by anacso17 — APPROVED** [review]

**Review by fernandohds564 — APPROVED** [review]

File: python_package/trackcpp.i [hunk 2]
```diff
@@ -36,6 +36,7 @@
 %include "std_string.i"
 %include "std_vector.i"
 %include "stl.i"
+%include "typemaps.i"
```
Comment (fernandohds564) [inline python_package/trackcpp.i:39]: what does this module do?

File: python_package/trackcpp.i [hunk 3]
```diff
@@ -97,6 +98,9 @@ double get_double_max() {
 %apply (double* INPLACE_ARRAY2, int DIM1, int DIM2 ) {
     (double *im_out, int n1_im_out, int n2_im_out)}
 
+// For kicktable_getkicks
+%apply double *OUTPUT { double &hkick__, double &vkick__ };
```
Comment (fernandohds564) [inline python_package/trackcpp.i:102]: Why is this needed? I thought we didn't need this type of declaration for python `float` type.

**Review by fernandohds564 — APPROVED** [review]

File: python_package/trackcpp.i (hunk 2 above)
Comment (xresende) [reply python_package/trackcpp.i:39]: oops... I did not noticed your msg here before. This module adds option to convert input/output function argument types.

File: python_package/trackcpp.i (hunk 3 above)
Comment (xresende) [reply python_package/trackcpp.i:102]: it converts input reference arguments (hkick, vkick) to output arguments, so that they ca be retrieved in python,
